index.html

```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
+
+<html lang="en">
+    <head>
+        <meta charset="UTF-8">
+        <meta name="viewport" content="width=device-width, initial-scale= 1.0">
+        <title> My portfolio </title>
+        <link href="styles/mainStyle.css" rel="stylesheet">
+    </head>
+    <body>
+        <header>
+            <nav class="toolbar">
+                <ul>
+                    <li class="textPlaceholder">  <p> John Doe </p> </li>
+                    <li> <button id="menu" type="button"> <img src="icons/menu.svg" alt=""> </button></li>
+                </ul>
+            </nav>
+        </header>
+        <section id="main" class="container">
+            <div class="container header">
+                <h1>
+                    Hey There. I’m Andrés
+                </h1>
+                <h3>
+                    I’am a Software Developer
+                </h3>
+                <p>
+                    I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+                    don’t hestiate to contact me.
+                </p>
+                <div class="container socialGroup">
+                    <button id="githubButton" type="button" class="social-media"><img src="icons/gitHub.svg" alt=""></button>
+                    <button id="linkedinButton" type="button" class="social-media"><img src="icons/linkedin.svg" alt=""></button>
+                    <button id="angelistButton" type="button" class="social-media"><img src="icons/angelist.svg" alt=""></button>
+                    <button id="twitterButton" type="button" class="social-media"><img src="icons/twitter.svg" alt=""></button>
+                    <button id="mediumButton" type="button" class="social-media"><img src="icons/medium.svg" alt=""></button>
+                </div>
```

Comment:
- [ ] Make sure the list of icons uses a `ul` tag to group them, and `li` per icon; each icon has a link (`a` tag). But instead, you used `div `and `button` so one can not click to reach your social media profiles.